samples/hello-spring-boot/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.spring.boot"
+version = "1.0.0-SNAPSHOT"
+
+plugins {
+    val kotlinVersion = "1.2.21"
+    id("org.springframework.boot") version "2.0.0.RELEASE"
+    id("org.jetbrains.kotlin.jvm") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.spring") version kotlinVersion
+    id("io.spring.dependency-management") version "1.0.4.RELEASE"
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs = listOf("-Xjsr305=strict")
+    }
+}
+
+val test by tasks.getting(Test::class) {
```

Comment:
Could be also `(tasks.findByName("test") as Test).useJUnitPlatform()`.
From my point of view it looks better... But don't know 🤷‍♂️